### PR TITLE
feat(images): upgrade icons to updated designs

### PR DESCRIPTION
**Pull request by Johannes-Andersen**
A lot of icons where outdated, low quality or was not loading. Should have fixed a few icons and moved them to svg for better quality

logo-purple.svg and logo-white.svg was massive for no reason. So made them just 30% of the orignial size with no visable changes﻿

### CODE REVIEW

File: src/technologies.json [hunk 1]
```diff
@@ -16656,7 +16660,7 @@
         16
       ],
       "html": "<style[^>]+[^<]+#cf-hcaptcha-container[^<]+</style>",
-      "icon": "hcaptcha-symbol-256.png",
+      "icon": "hCaptcha.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:16663]: Screenshot 2020-11-08 at 16 05 04

File: src/technologies.json [hunk 2]
```diff
@@ -17415,7 +17419,7 @@
         "<link[^>]+recaptcha",
         "<div[^>]+class=\"g-recaptcha\""
       ],
-      "icon": "reCAPTCHA.png",
+      "icon": "reCAPTCHA.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:17422]: Screenshot 2020-11-08 at 16 05 54

File: src/technologies.json [hunk 3]
```diff
@@ -14949,7 +14949,11 @@
       ],
       "description": "Vimeo is a video hosting, sharing and services platform. Vimeo operation an ad-free basis by providing subscription plans.",
       "html": "(?:<(?:param|embed)[^>]+vimeo\\.com/moogaloop|<iframe[^>]player\\.vimeo\\.com)",
-      "icon": "Vimeo.png",
+      "js": {
+        "Vimeo.Player": "",
+        "VimeoPlayer": ""
+      },
+      "icon": "Vimeo.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:14956]: Screenshot 2020-11-08 at 16 07 10

File: src/technologies.json [hunk 4]
```diff
@@ -14354,7 +14354,7 @@
         10
       ],
       "description": "TrackJS is an error monitoring agent for production web sites and applications.",
-      "icon": "TrackJs.png",
+      "icon": "TrackJs.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:14357]: Screenshot 2020-11-08 at 16 07 47

File: src/technologies.json [hunk 5]
```diff
@@ -14122,7 +14122,7 @@
         68
       ],
       "description": "TextHelp is a literacy, accessibility and dyslexia software developer for people with reading and writing difficulties.",
-      "icon": "Texthelp.png",
+      "icon": "Texthelp.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:14125]: Screenshot 2020-11-08 at 16 08 19

File: src/technologies.json [hunk 6]
```diff
@@ -6698,7 +6698,7 @@
       "cats": [
         16
       ],
-      "icon": "Imperva.svg",
+    "icon": "Imperva.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:6701]: Screenshot 2020-11-08 at 16 09 11

File: src/technologies.json [hunk 7]
```diff
@@ -6503,7 +6503,7 @@
         10
       ],
       "description": "Hotjar is a suite of analytic tools to assist in the gathering of qualitative data, providing feedback through tools such as heatmaps, session recordings, and surveys.",
-      "icon": "Hotjar.png",
+      "icon": "Hotjar.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:6506]: Screenshot 2020-11-08 at 16 10 17

File: src/technologies.json [hunk 8]
```diff
@@ -5983,7 +5983,7 @@
         "googletagmanager\\.com/ns\\.html[^>]+></iframe>",
         "<!-- (?:End )?Google Tag Manager -->"
       ],
-      "icon": "Google Tag Manager.png",
+      "icon": "Google Tag Manager.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:5986]: Screenshot 2020-11-08 at 16 10 32

File: src/technologies.json [hunk 9]
```diff
@@ -6335,7 +6335,7 @@
       "cats": [
         10
       ],
-      "icon": "Heap.png",
+      "icon": "Heap.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:6338]: Screenshot 2020-11-08 at 16 11 09

File: src/technologies.json [hunk 10]
```diff
@@ -5924,7 +5924,7 @@
         "X-Mod-Pagespeed": "([\\d.]+)\\;version:\\1",
         "X-Page-Speed": "(.+)\\;version:\\1"
       },
-      "icon": "Google PageSpeed.png",
+      "icon": "Google PageSpeed.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:5927]: Screenshot 2020-11-08 at 16 11 52

File: src/technologies.json [hunk 11]
```diff
@@ -5895,7 +5895,7 @@
         35
       ],
       "description": "Google Maps is a web mapping service. It offers satellite imagery, aerial photography, street maps, 360° interactive panoramic views of streets, real-time traffic conditions, and route planning for traveling by foot, car, bicycle and air, or public transportation.",
-      "icon": "Google Maps.png",
+      "icon": "Google Maps.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:5898]: Screenshot 2020-11-08 at 16 13 19

File: src/technologies.json [hunk 12]
```diff
@@ -5839,7 +5839,7 @@
       "headers": {
         "Server": "Google Frontend"
       },
-      "icon": "Google App Engine.png",
+      "icon": "Google App Engine.svg",
```
Comment (Johannes-Andersen) [inline src/technologies.json:5842]: Screenshot 2020-11-08 at 16 14 04

File: src/drivers/webextension/images/logo-purple.svg [hunk 13]
```diff
@@ -1,29 +1,9 @@
...
-                    <path d="M255.7556,11.5484 L255.7556,19.3924 C255.0966,19.3474 254.4146,19.3474 254.0506,19.3474 C250.5946,19.3474 248.8206,21.5294 248.8206,24.5084 L248.8206,34.7404 L241.5446,34.7404 L241.5446,11.9574 L248.8206,11.9574 L248.8206,15.0044 L248.9116,15.0044 C250.6396,12.7304 253.0046,11.5254 255.5056,11.5254 C255.5736,11.5254 255.6876,11.5254 255.7556,11.5484" id="Fill-17"></path>
-                    <polygon id="Fill-3" points="47.1903 1.6579 36.8453 34.7399 29.7283 34.7399 24.1803 12.9809 24.1123 12.9809 18.3603 34.7399 11.2433 34.7399 0.8983 1.6579 9.1973 1.6579 14.7903 21.9619 14.8583 21.9619 19.9743 1.6579 28.1823 1.6579 33.2303 21.9619 33.2983 21.9619 38.8913 1.6579"></polygon>
-                </g>
-            </g>
-        </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="389" height="97">
```
Comment (Johannes-Andersen) [inline src/drivers/webextension/images/logo-purple.svg:1]: Screenshot 2020-11-08 at 16 14 43

File: src/drivers/webextension/images/logo-white.svg [hunk 14]
```diff
@@ -1,29 +1,15 @@
...
-                    <path d="M255.7556,11.5484 L255.7556,19.3924 C255.0966,19.3474 254.4146,19.3474 254.0506,19.3474 C250.5946,19.3474 248.8206,21.5294 248.8206,24.5084 L248.8206,34.7404 L241.5446,34.7404 L241.5446,11.9574 L248.8206,11.9574 L248.8206,15.0044 L248.9116,15.0044 C250.6396,12.7304 253.0046,11.5254 255.5056,11.5254 C255.5736,11.5254 255.6876,11.5254 255.7556,11.5484" id="Fill-17"></path>
-                    <polygon id="Fill-3" points="47.1903 1.6579 36.8453 34.7399 29.7283 34.7399 24.1803 12.9809 24.1123 12.9809 18.3603 34.7399 11.2433 34.7399 0.8983 1.6579 9.1973 1.6579 14.7903 21.9619 14.8583 21.9619 19.9743 1.6579 28.1823 1.6579 33.2303 21.9619 33.2983 21.9619 38.8913 1.6579"></polygon>
-                </g>
-            </g>
-        </g>
+<svg xmlns="http://www.w3.org/2000/svg" width="389" height="97">
```
Comment (Johannes-Andersen) [inline src/drivers/webextension/images/logo-white.svg:1]: Screenshot 2020-11-08 at 16 15 08

**Comment by AliasIO** [comment]
Nice one, thanks!

**Pull request merged**